compiler/src/dotty/tools/dotc/semanticdb/ExtractSemanticDB.scala

```diff
@@ -241,8 +242,13 @@ class ExtractSemanticDB extends Phase:
               if imported != nme.WILDCARD then
                 for alt <- tree.expr.tpe.member(imported).alternatives do
                   registerUseGuarded(None, alt.symbol, sel.imported.span, tree.source)
-                  if (alt.symbol.companionClass.exists)
-                    registerUseGuarded(None, alt.symbol.companionClass, sel.imported.span, tree.source)
+                  try
+                    if (alt.symbol.companionClass.isDefinedInCurrentRun)
```

Comment:
Why was this changed to `.isDefinedInCurrentRun` from `.exists`? the test case `pos/i13532` still passes if i change it back, and then the semanticdb expect files do not need to change

Reply:
resolved by newer commits